PORTAL/jobs/__main__.py

```diff
@@ -669,6 +669,11 @@ def configure_root_logger(
         *,
         maxlevel: int = logging.CRITICAL,
 ) -> None:
+    # pytest does its own monkey-patching of logging that isn't compatible with
+    # this.
+    if "pytest" in sys.modules:
+        return
+
```

Comment:
Would it make sense to move this down to right after `#logger.propagate = False`?  Then CLI usage would have things like `-v` respected properly.  That said, we probably also want the formatter applied as well.

Reply:
Yeah, that works.  Setting the formatter is a bit trickier, because we'd need to set it on pytest's handler.  Probably doable, but maybe save that until we want to test specific logging messages.

Reply:
Yeah, let's punt on that.  It mostly only matters if we're checking exactly what a command prints to stdout (via the logger).